packages/xo-web/src/common/xo/index.js

```diff
@@ -1238,14 +1238,23 @@ export const migrateVm = (vm, host) =>
   confirm({
     title: _('migrateVmModalTitle'),
     body: <MigrateVmModalBody vm={vm} host={host} />,
-  }).then(params => {
+  }).then(async params => {
     if (!params.targetHost) {
       return error(
         _('migrateVmNoTargetHost'),
         _('migrateVmNoTargetHostMessage')
       )
     }
-    return _call('vm.migrate', { vm: vm.id, ...params })
+
+    try {
+      await _call('vm.migrate', { vm: vm.id, ...params })
+    } catch (error) {
```

Comment:
Maybe a notification with the error. The user need to know the error before force migrating the VM.